### ISSUE TITLE
data-binding raises TypeError

### ISSUE THREAD

**Issue by harvimt** [Prezent/ractive-decorators-select2#17]
`TypeError: this.callback.call is not a function`

I'm using browserify and I get this in both firefox and chromium for Mac (haven't tested other platforms)

I've tried to keep this as close to the demo code as possible.

index.html

``` html
<html>
<head>
    <meta charset="utf-8">
    <link rel="stylesheet" href="node_modules/select2/dist/css/select2.css">
<body>
    <div id="demo"></div>
    <script type="application/ractive" id="demo-template">
        <select decorator='select2:demo' value='{{selected}}'>
            {{#options}}
                <option value='{{.}}'>{{.}}</option>
            {{/options}}
        </select>
        <button on-click='reset-data'>Reset (two-way binding)</button>
        <p>Selected option: {{selected}}</p>
    </script>
    <script src="bundle.js"></script>
```

index.js

``` javascript
var $ = require('jquery');
var Ractive = require('ractive');
require('select2')
require('ractive-decorators-select2');

// define select2 options
Ractive.decorators.select2.type.demo = function (node) {
    return {
        width: '25%'
    }
};

ractive = new Ractive({
    el: 'demo',
    template: '#demo-template',
    data: {
        selected: 'one',
        options: [ 'one', 'two', 'three', 'four', 'five', 'six' ]
    }
});

ractive.on('reset-data', function(event) {
    this.set('selected', 'one');
});
```

package.json

``` json
{
  "name": "sel2-ractive-minimal",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "jquery": "=2.1.4",
    "ractive": "^0.7.3",
    "ractive-decorators-select2": "^0.1.4",
    "select2": "=4.0.0"
  }
}
```

browserify command: `browserify -o bundle.js index.js`

**Comment by harvimt** [comment]
I only get the error if I use data-binding, if I leave off the `value="{{selected}}"` the error goes away.

**Comment by harvimt** [comment]
Also worth noting that I don't get this error on your demo page, so I think it might be a problem with how this works with `browserify`

**Comment by sandermarechal** [comment]
Yes, it appears to be an issue with two-way binding using Ractive 0.7.x. When I use Ractive 0.6.x it works fine. Using browserify or not does not matter in my tests.

I'll look into this further

**Comment by sandermarechal** [comment]
This was already resolved in the latest 0.1.7 version. I forgot to publish that to NPM however. Fixed.
